README.md

```diff
@@ -827,12 +827,12 @@ Returns: `string`
 
 Usage:
 ```
-{{currency 1000000 code='USD'}}  => $1,000,000.00
-{{currency 1000000 code='EUR'}}  => 1 000 000,00 €
-{{currency 1000000 code='EUR' precision=0}}  => 1 000 000 €
+{{formatCurrency 1234567.89 currency='USD'}}  => $1,234,567.89
```

Comment:
Okay, how about doing something like this:

```javascript
let params = Object.assign({}, args, {
    currency: code
});
// Send params to the plugin's function
```
And we can pass `code` instead of `currency` like this:
```
{{currency 1000000 code='EUR' precision=0}}
```